src/lib/styles/themes/dark.scss

```diff
@@ -161,56 +163,56 @@
 
   /* Light/Text/Text-xx? */
   --text-color: var(--neutral-50);
-  --text-color-rgb: 240, 237, 245;
+  --text-color-rgb: 255, 255, 255;
 
   /* Design: Dark/Bg/Background-03 */
-  --input-background: var(--violet-450);
-  --input-background-contrast: var(--text-color);
+  --input-background: var(--night-500);
+  --input-background-contrast: var(--neutral-50);
 
   /* Design: Dark/Bg/Background */
-  --focus-background: var(--violet-500);
-  --focus-background-rgb: 61, 42, 92;
-  --focus-background-contrast: var(--text-color);
-  --focus-background-tint: var(--violet-400);
+  --focus-background: var(--night-575);
+  --focus-background-rgb: 33, 39, 51;
+  --focus-background-contrast: var(--neutral-50);
+  --focus-background-tint: var(--night-400);
 
   /* Design: Dark/Bg/Background-01 */
-  --card-background: var(--violet-500);
-  --card-background-rgb: 61, 42, 92;
-  --card-background-contrast: var(--text-color);
-  --card-background-contrast-rgb: var(--text-color-rgb);
-  --card-background-shade: var(--violet-500);
-  --card-background-tint: var(--violet-400);
-  --card-background-disabled: var(--cp-dark-1000-20);
+  --card-background: var(--night-500);
+  --card-background-rgb: 43, 51, 67;
+  --card-background-contrast: var(--neutral-50);
+  --card-background-contrast-rgb: 255, 255, 255;
+  --card-background-shade: var(--night-575);
+  --card-background-tint: var(--night-400);
+  --card-background-disabled: var(--neutral-blue-a10);
```

Comment:
I noticed very low contrast on the neuron badges (on the left) and disabled buttons (on the right).
Please work with Artem on these before merging.
![image](https://github.com/user-attachments/assets/01a01800-aba7-4a3f-9c39-ead896957789)

Reply:
Thanks, updated
<img width="808" alt="image" src="https://github.com/user-attachments/assets/cd57f9e4-3de5-4db9-8c14-1e8ed15779bd">

Reply:
I mentioned 2 things and it looks like you changed 7 things.
What are the other 5?

Reply:
Requests from Artem. I committed them separately, to keep it clear.
1. "Increase icons on card contrast" (you mentioned)
2. "Swap modal backgrounds" to be same as in Figma and avoid having contrast issue after the next step (3.).
3. "Increase disabled button contrast" (you mentioned).
4. "Increase scrollbar contrast" Artem mentioned.
5. "Make the disabled input state more distinguishable" Artem mentioned.